main.py

```diff
@@ -201,7 +178,7 @@ def _load_switch(self, switch_id, switch_att):
         switch.description['software'] = switch_att.get('software')
         switch.description['serial'] = switch_att.get('serial', '')
         switch.description['data_path'] = switch_att.get('data_path', '')
-        self.update_instance_metadata(switch)
+        switch.extend_metadata(switch_att["metadata"])
```

Comment:
The database models have unified `metadata` accordingly so now it's simpler to load.